deploy/webhooks/agentclusterinstall-webhook.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: agentclusterinstallvalidators.admission.agentinstall.openshift.io
+webhooks:
+- name: agentclusterinstallvalidators.admission.agentinstall.openshift.io
+  clientConfig:
+    service:
+      # reach the webhook via the registered aggregated API
+      namespace: default
```

Comment:
These seem unlikely to be what we want. Should they be changed?

Reply:
This is only for the subsystem tests. Deploying this via the operator is covered in https://issues.redhat.com/browse/MGMT-7147